README.md

```diff
@@ -488,15 +488,19 @@ attributes:
 
 -   **expression**: indicates that the value of the target attribute will not be
     the plain value or the measurement, but an expression based on a combination
-    of the reported values. See the
-    [Expression Language definition](doc/expressionLanguage.md) for details
+    of the reported values. IoT Agent Lib supports two expression languages:
```

Comment:
```suggestion
    of the reported values. IoT Agent Lib supports two expression languages (which one is used is explained in [expression plugin documentation](expressionLanguage.md)):
```

to include a link where how to choose between one language or other is explained.

Reply:
will fix

Reply:
i am afraid that this cannot be fixed due to recent updates to the README.md by @jason-fox

Reply:
Thus, NTC.